style.css

```diff
@@ -103,7 +103,6 @@
 .menuPage {
```

Comment:
The design of your mobile menu is not matching exactly with the Figma template of your choosing. As you can see, each link along with the bottom border, has some margin on the left and right side. Kindly replicate the same in your design. The requirement of this module is to make a pixel prefect replica. 

| Your design  | Figma design |
| ------------- | ------------- |
| ![image](https://github.com/bhushan354/My-Portfolio/assets/33268927/d598694f-85f7-472d-9721-ec27d78dc75c) | ![image](https://github.com/bhushan354/My-Portfolio/assets/33268927/5f0062e2-8101-4025-996d-e5d1ed67d144) |